src/hydra-queue-runner/build-remote.cc

```diff
@@ -286,9 +286,6 @@ static BuildResult performBuild(
     counter & nrStepsBuilding
 )
 {
-
-    BuildResult result;
```

Comment:
Just moving this down, it is for the response so it doesn't need to be in scope for the request.